protos/world.proto

```diff
@@ -10,4 +10,10 @@ message GetAirbasesRequest {
 
 message GetAirbasesResponse {
   repeated Airbase airbases = 1;
+}
+
+message GetMarkPanelsRequest {}
```

Comment:
Would it make sense to rename `EmptyResponse` to `Empty` and re-use it for empty responses too?

Reply:
So I have been reading gRPC best practices as created by teams that have been using gRPC for years and their recommendations can be summed up as `SHARE NOTHING".

Their recommendation is that if you have a Service with a method called `GetMarkPanels` then your request and response will be exactly that name with the `Request` and `Response` suffixes.

This greatly improves backwards compatibility and autogeneration / documentation possibilities. While I don't think this is a huge benefit for _us_ since we are unlikely to be changing these APIs I am loath to go against these recommendations.

Due to this I am thinking that after this commit and your Unit streaming commit I will need to go through everything and changed the requests and responses if we want to follow the recommendations.

What do you think?

* https://medium.com/@akhaku/protobuf-definition-best-practices-87f281576f31
* https://docs.buf.build/style-guide/
* https://docs.buf.build/lint-rules#rpc_request_standard_name-rpc_response_standard_name-rpc_request_response_unique

Reply:
I am totally up for following best practices, thanks for going through them 👍